src/coreclr/src/jit/lower.cpp

```diff
@@ -3190,7 +3211,7 @@ void Lowering::LowerRetStruct(GenTreeUnOp* ret)
 //
 void Lowering::LowerRetStructLclVar(GenTreeUnOp* ret)
 {
-    assert(!comp->compMethodReturnsMultiRegRetType());
+    assert(!comp->compMethodReturnsResInMultiplyRegisters());
```

Comment:
This was already the case, but there should probably be a comment in the function header that this method is only for single-register struct returns.